epub33/core/index.html

```diff
@@ -1937,7 +1937,7 @@ <h4>URLs in the OCF abstract container</h4>
 						<ul>
 							<li>relative URL strings starting with a <code>/</code> (<code>U+002F</code>) (for example,
 									<code>/EPUB/content.xhtml</code>) are disallowed; </li>
-							<li>relative URL strings containing more [=double-dot path segments=] than needed to reach
+							<li>relative URL strings containing more [=double-dot URL path segments=] than needed to reach
```

Comment:
```suggestion
							<li>relative URL strings containing more [=double-dot URL path segments=] than needed to reach
```

It seems that xref does not find this expression. Has it been added to [xref](https://respec.org/xref/)?

Reply:
(I presume that also explains why the check below failed.)

Reply:
I suppose you might have to wait a couple hours up to a full day for it to be updated in Reffy and then re-run CI.

Reply:
Ah, o.k., I did not realize that change is so recent. No problems, we should indeed wait then. Thx.

Reply:
xref is o.k. now, so I will merge this. Thanks @annevk